packages/google_sign_in/google_sign_in/android/gradle.properties

```diff
@@ -1,4 +1,3 @@
 org.gradle.jvmargs=-Xmx1536M
-android.enableR8=true
 android.useAndroidX=true
 android.enableJetifier=true
```

Comment:
You can probably remove all of these, and just leave "org.gradle.jvmargs=-Xmx1536M"

Reply:
@blasten What happens if someone builds the plugin in an app that's using Gradle 3.3 or earlier?

Reply:
These files are actually *not* relevant in Flutter. The app's gradle.properties is the one that matters in a Flutter app.

Reply:
If a plugin is packaged as an AAR artifact then this file is relevant, but there's no reason for doing that in Flutter.

Reply:
Should we just nuke all of these files from the repo then? (And maybe [the template](https://github.com/flutter/flutter/blob/master/packages/flutter_tools/templates/plugin/android.tmpl/gradle.properties.tmpl)? Or is that to future-proof against maybe supporting AAR?)

Reply:
Yes, we should remove it. webview is a good example for what the android folder should have: https://github.com/flutter/plugins/tree/master/packages/webview_flutter/webview_flutter/android

Reply:
Thanks, filed https://github.com/flutter/plugins/pull/4309